debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/processor/AbstractLogMinerEventProcessor.java

```diff
@@ -460,8 +462,9 @@ protected void addToTransaction(String transactionId, LogMinerEventRow row, Supp
         if (isTransactionIdAllowed(transactionId)) {
             Transaction transaction = getTransactionCache().get(transactionId);
             if (transaction == null) {
-                LOGGER.trace("Transaction {} not in cache, creating.", transactionId);
+                LOGGER.debug("Transaction {} not in cache for DML, creating.", transactionId);
```

Comment:
Is there a reason why we moved from `TRACE` to `DEBUG` log level here?

Reply:
@Naros  When we running into this line,  it means we lost the transaction start event, right? The transaction will not have userName in this case. We need to log this, so we can figure out why the user name filter doesn't filter this transaction.

Reply:
I've pushed a commit to change this back to `TRACE` level as I don't see a need for this to be `DEBUG`.